docs/features/CONCEPTS.md

```diff
@@ -0,0 +1,132 @@
+---
+title: The Hugging Face Concepts Library and Importing Textual Inversion files
+---
+
+# :material-file-document: Concepts Library
+
+## Using Textual Inversion Files
+
+Textual inversion (TI) files are small models that customize the output of
+Stable Diffusion image generation. They can augment SD with
+specialized subjects and artistic styles. They are also known as
+"embeds" in the machine learning world.
+
+Each TI file introduces one or more vocabulary terms to the SD
+model. These are known in InvokeAI as "triggers." Triggers are often,
+but not always, denoted using angle brackets as in
+"&lt;trigger-phrase&gt;". The two most common type of TI files that you'll
+encounter are `.pt` and `.bin` files, which are produced by different
+TI training packages. InvokeAI supports both formats, but its [built-in
```

Comment:
Might be worth calling "Creating your own TI files" as its own H2. People might miss this on a scan

Reply:
I'll do that on a subsequent documentation PR. Can't do that on this one without your having to re-review!